src/ai/backend/manager/scheduler/dispatcher.py

```diff
@@ -242,14 +244,22 @@ async def _load_scheduler(
         db_conn: SAConnection,
         sgroup_name: str,
     ) -> AbstractScheduler:
-        query = (
-            sa.select([scaling_groups.c.scheduler])
-            .select_from(scaling_groups)
-            .where(scaling_groups.c.name == sgroup_name)
-        )
-        result = await db_conn.execute(query)
-        scheduler_name = result.scalar()
-        return load_scheduler(scheduler_name, self.shared_config['plugins']['scheduler'])
+        try:
+            query = (
+                sa.select([scaling_groups.c.scheduler, scaling_groups.c.scheduler_opts])
+                .select_from(scaling_groups)
+                .where(scaling_groups.c.name == sgroup_name)
+            )
+            result = await db_conn.execute(query)
+            row = result.one()
+            scheduler_name = row['scheduler']
+            scheduler_opts = row['scheduler_opts']
+        except MultipleResultsFound as e:
+            log.exception('_load_scheduler(): multiple scheduler! \n{}', repr(e))
```

Comment:
When logging exceptions inside the `except` block, you don't have to manually serialize the exception.
```suggestion
            log.exception("_load_scheduler(): multiple scheduler!")
```
will do the what you have intendend.
In Python's `except` blocks, the runtime automatically sets `sys.exc_info()` tuple and most exception-handling utilities including loggers are able to recognize it.
And although this is not a strict rule yet, please use double-quotes for user-facing strings. In the future we may migrate to use the black style and it's worth to try minimizing diffs for new codes.